docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -82,22 +82,46 @@ You can configure one or more distribution servers for Galaxy in your :file:`ans
 
    [galaxy_server.release_galaxy]
    url=https://galaxy.ansible.com/
-   token=my_token
+   token=abcdefghijklmnopqrtuvwxyz
 
-See :ref:`galaxy_server_config` for complete details.
+You cannot use ``apt-key`` with any servers defined in your :ref:`galaxy_server_list <galaxy_server_config>`. See :ref:`galaxy_server_config` for complete details.
+
+.. _galaxy_use_token_arg:
+
+Specifying the token at the command line
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can specify the API token at the command line using the ``--token`` argument of the :ref:`ansible-galaxy` command. There are three ways to specify the distribution server when passing the token at the command line:
+
+* using the ``--server`` argument of the :ref:`ansible-galaxy` command
+* relying on the default (https://galaxy.ansible.com)
+* setting a server in configuration by creating a :ref:`GALAXY_SERVER` setting in your :file:`ansible.cfg` file
+
+For example:
+
+.. code-block:: bash
+
+     ansible-galaxy collection publish path/to/my_namespace-my_collection-1.0.0.tar.gz --token abcdefghijklmnopqrtuvwxyz
+
+.. warning::
+
+   Using the ``--token`` argument is insecure. Passing secrets at the command line may expose them to others on the system.
 
 .. _building_collections:
 
-Building a collection tarball
-=============================
+Building your collection tarball
+================================
 
-Once you have configured one or more distribution servers, you must build a collection tarball. To build a collection, run ``ansible-galaxy collection build`` from inside the root directory of the collection:
+After you configure one or more distribution servers, you must build a collection tarball. To build a collection tarball:
```

Comment:
```suggestion
After configuring one or more distribution servers, build a collection tarball. The following procedure builds a collection tarball.
```
Keeping the active voice and consistency with an earlier review comment.

Reply:
Fixed in 2b84951fc61422551f159ccf5df686beff5fb23e.